src/openrct2/peep/Peep.cpp

```diff
@@ -3078,7 +3078,8 @@ void Peep::PerformNextAction(uint8_t& pathing_result, TileElement*& tile_result)
     }
 
     auto newLoc = *loc;
-    if ((newLoc.x & 0xFFE0) == next_x && (newLoc.y & 0xFFE0) == next_y)
+    CoordsXY centeredNewLoc = newLoc.ToTileStart();
```

Comment:
You call the variable `centered`, but it isn't centred. `truncatedNewLoc` is a better name IMO.

(Also note that in `centered` is American English and that it's `centred` in British English, which is used for OpenRCT2.)

Reply:
I did know that, because I proposed `ToTileCentre`, just failed here hahaha